web/gui/dashboard_info.js

```diff
@@ -960,6 +960,10 @@ netdataDashboard.context = {
         info: 'CPU softirqs in detail. At the <a href="#menu_cpu">CPUs</a> section, softirqs are analyzed per CPU core.'
     },
 
+    'system.softirq_latency': {
+        info: 'Total time spent servicing software interrupts.'
```

Comment:
I did not review this PR yet, but while I am doing tests to verify that eBPF program is running on different kernels, it called my attention that we are not saying that the new charts is coming from `eBPF.plugin`, please, add a reference to the plugin here.

This is something already discussed with @hugovalente-pm and @manos-saratsis and it is one of the biggest motivations for me work in this [issue](https://github.com/netdata/netdata/issues/11439).

Thank you!

Reply:
Added in https://github.com/netdata/netdata/pull/11445/commits/75edc650dc3653a2c5f4197a932c113524a97902